File: Convert_String.java
/*
*Solution By : Nimish Shandilya
*LinkedIn : https://www.linkedin.com/in/nimish-shandilya-0a429b22b
*
*Problem : Convert String
*Problem Source : Coding Ninjas
* 
*/

public class Convert_String{
	public static String convertString(String str) {
        // Write your code here
        str=" "+str;
        String x="";
        for(int i=0;i<str.length();i++){
            if(str.charAt(i)==' '&&i!=str.length()-1){
                x=x+" "+(str.charAt(i+1)+"").toUpperCase();
                i++;
            }
            else{
                x=x+str.charAt(i);
            }
        }
        return x.substring(1,x.length());
    }

}

File: Convert_String.py
'''
*Solution By : Nimish Shandilya
*LinkedIn : https://www.linkedin.com/in/nimish-shandilya-0a429b22b
*
*Problem :  Convert String
*Problem Source : Coding Ninjas
* 
'''

from os import *
from sys import *
from collections import *
from math import *

def convertString(s):
    # Write your code here
    a=""
    for i in s.split():
        a=a+i[:1].upper()+i[1:]+" "
    return a[:len(a)-1]
